charts/scalardb/values.yaml

```diff
@@ -19,7 +19,7 @@ envoy:
     # -- admin log path
     adminAccessLogPath: /dev/stdout
     # -- list of service name and port
-    serviceListeners: scalardb-service:50051,scalardb-privileged:50052
+    serviceListeners: scalardb-service:60051
```

Comment:
Scalar DB doesn't have **privileged port** like Scalar DL.
So, we need the `scalardb-service:60051` only.